### DISCUSSION TITLE
Ability to change an apps region

### DISCUSSION THREAD

**Discussion by codewithtyler** [supabase/supabase#31295] [Feature Requests]
As far as I know the only way to change a region is to go and create the app in the new region and then run queries and migration scripts to reseed the database in the new app. Sadly this doesn't account for migrating auth users. It would be great if each app had the ability for us to change the region the app is located. Maybe the app goes offline for 5-10 minutes while that happens that would be fine so long as we can change the region when we accidentally set it incorrectly

**Comment by GaryAustin1** [comment]
Not that your feature request would not be nice...  But the migration scripts do move the auth users.   https://supabase.com/docs/guides/platform/migrating-within-supabase

**Comment by codewithtyler** [comment]
Ah I wasn't aware of that . That's good to know. However, I still thinks this feature would be something that's good to have as with the addition of Bolt <> Supabase integration the majority of the Bolt users are not as technically savvy as you or I and may not have the technical expertise to do this manually which is why I was suggesting the ability to do this from the app itself.
